src/work-allocation/services/work-allocation-task.service.ts

```diff
@@ -14,7 +15,14 @@ export class WorkAllocationTaskService {
    * @param taskId specifies which task should be completed.
    */
   public completeTask(taskId: string): Observable<any> {
-    // Make a POST with an empty payload.
     return this.http.post<any>(`${BASE_URL}${taskId}/complete`, {});
   }
+
+  public postTask(task: TaskSearchParameters): Observable<any> {
+    return this.http.post<any>(`${BASE_URL}`, {});
```

Comment:
Doesn't this need to pass the search parameters in the body?

Reply:
yes